virtual_rainforest/models/animals/constants.py

```diff
@@ -16,11 +16,35 @@ class Term:
     """Handles the coefficient of scaling equations"""
 
 
-"""From (Damuth 1987)."""
-DamuthsLaw = Term(-0.75, 4.23)
+"""Mammalian herbivore population density, observed allometry (Damuth 1987). [kg]"""
+DamuthsLawTerms = ScalingTerms(-0.75, 4.23)
+
+"""Temporary basal metabolic rate values for mammals (citation from Rallings). [g]"""
+MetabolicRateTerms = ScalingTerms(0.75, 0.047)
+
+"""Scaling of mammalian herbivore fat mass (citation from Rallings). [g]"""
+FatMassTerms = ScalingTerms(1.19, 0.02)
+
+"""Scaling of mammalian herbivore muscle mass (citation from Rallings).[g]"""
+MuscleMassTerms = ScalingTerms(1.0, 0.38)
+
+"""Mammalian maximum intake rate (g/min) from (Shipley 1994)."""
+IntakeRateTerms = ScalingTerms(0.71, 0.63)
+
+
+@dataclass
+class Term:
```

Comment:
I think I miss the point of this. If it is just one number, why not just setting it directly:

```python
ConversionEfficiency: float = 0.1
...
```

Also, the convention in python is using UPPERCASE for constants. I'm surprised `flake8` does not complain about you using CamelCase.

Reply:
I have things set up as classes because they will end up being quite a bit more complex in the not-so-distant future. Both the scaling relationships and the single constants will end up taking information about the animal functional type, environmental conditions, and/or food types to sort out the specific final values. I imagined this as making the most sense as a class but I'm open to alternatives.

Reply:
Ah, ok. In that case, it makes sense as it is now. Maybe add a `ToDo` to the docstring of the class explaining this.